Galeri: build fails with Epetra disabled

## Bug Report
@trilinos/galeri 

With `Trilinos_ENABLE_Epetra:BOOL=OFF` defined, I get the following build errors:

```
packages/galeri/src-xpetra/../src-epetra/Galeri_Maps.h:47:10: fatal error: Epetra_ConfigDefs.h: No such file or directory
 #include "Epetra_ConfigDefs.h"
```

Full configure script:

```
/projects/sems/install/rhel6-x86_64/sems/utility/cmake/3.12.2/bin/cmake \
  -G "Ninja" \
  -D Trilinos_CONFIGURE_OPTIONS_FILE:FILEPATH=$TRILINOS_DIR/cmake/std/PullRequestLinuxCommonTestingSettings.cmake \
  -D Trilinos_ENABLE_Tpetra:BOOL=ON \
  -D Trilinos_ENABLE_Epetra:BOOL=OFF \
  -D Trilinos_ENABLE_Moertel:BOOL=OFF \
  -D Trilinos_ENABLE_ShyLU_DDFROSch:BOOL=OFF \
  -D Trilinos_ENABLE_Stokhos:BOOL=OFF \
  -D ROL_ENABLE_EXAMPLES:BOOL=OFF \
  -D Tpetra_ENABLE_DEPRECATED_CODE:BOOL=OFF \
  -D Xpetra_ENABLE_DEPRECATED_CODE:BOOL=OFF \
  $TRILINOS_DIR
```

--- a/packages/galeri/example-fem/CMakeLists.txt
+++ b/packages/galeri/example-fem/CMakeLists.txt
@@ -1,20 +1,21 @@
 
-TRIBITS_ADD_EXECUTABLE(
-  Grid
-  SOURCES Grid.cpp
-  COMM serial mpi
-  )
+IF(${PACKAGE_NAME}_ENABLE_Epetra)
+  TRIBITS_ADD_EXECUTABLE(
+    Grid
+    SOURCES Grid.cpp
+    COMM serial mpi
+    )
 
-TRIBITS_ADD_EXECUTABLE(
-  Laplacian3D
-  SOURCES Laplacian3D.cpp
-  COMM serial mpi
-  )
+  TRIBITS_ADD_EXECUTABLE(
+    Laplacian3D
+    SOURCES Laplacian3D.cpp
+    COMM serial mpi
+    )
 
-TRIBITS_ADD_EXECUTABLE(
-  AdvDiff2D
-  SOURCES AdvDiff2D.cpp
-  COMM serial mpi
-  )
-
+  TRIBITS_ADD_EXECUTABLE(
+    AdvDiff2D
+    SOURCES AdvDiff2D.cpp
+    COMM serial mpi
+    )
+ENDIF()
 

--- a/packages/galeri/example/CMakeLists.txt
+++ b/packages/galeri/example/CMakeLists.txt
@@ -1,60 +1,62 @@
 
-IF (NOT Trilinos_NO_32BIT_GLOBAL_INDICES)
+IF(${PACKAGE_NAME}_ENABLE_Epetra)
+  IF (NOT Trilinos_NO_32BIT_GLOBAL_INDICES)
 
-  TRIBITS_ADD_EXECUTABLE_AND_TEST(
-    Map
-    SOURCES Map.cpp
-    NUM_MPI_PROCS 1
-    COMM serial mpi
-    )
+    TRIBITS_ADD_EXECUTABLE_AND_TEST(
+      Map
+      SOURCES Map.cpp
+      NUM_MPI_PROCS 1
+      COMM serial mpi
+      )
 
-  TRIBITS_ADD_EXECUTABLE_AND_TEST(
-    CrsMatrix
-    SOURCES CrsMatrix.cpp
-    NUM_MPI_PROCS 1
-    COMM serial mpi
-    )
+    TRIBITS_ADD_EXECUTABLE_AND_TEST(
+      CrsMatrix
+      SOURCES CrsMatrix.cpp
+      NUM_MPI_PROCS 1
+      COMM serial mpi
+      )
 
-  TRIBITS_ADD_EXECUTABLE_AND_TEST(
-    VbrMatrix
-    SOURCES VbrMatrix.cpp
-    NUM_MPI_PROCS 1
-    COMM serial mpi
-    )
+   TRIBITS_ADD_EXECUTABLE_AND_TEST(
+      VbrMatrix
+      SOURCES VbrMatrix.cpp
+      NUM_MPI_PROCS 1
+      COMM serial mpi
+      )
 
-  TRIBITS_ADD_EXECUTABLE_AND_TEST(
-    LinearProblem
-    SOURCES LinearProblem.cpp
-    NUM_MPI_PROCS 1
-    COMM serial mpi
-    )
+    TRIBITS_ADD_EXECUTABLE_AND_TEST(
+      LinearProblem
+      SOURCES LinearProblem.cpp
+      NUM_MPI_PROCS 1
+      COMM serial mpi
+      )
 
-ENDIF()
+  ENDIF()
 
-IF (NOT Trilinos_NO_64BIT_GLOBAL_INDICES)
+  IF (NOT Trilinos_NO_64BIT_GLOBAL_INDICES)
 
-  TRIBITS_ADD_EXECUTABLE_AND_TEST(
-    Map_LL
-    SOURCES Map.cpp
-    NUM_MPI_PROCS 1
-    COMM serial mpi
-	TARGET_DEFINES -DGALERI_TEST_USE_LONGLONG_GO
-    )
+    TRIBITS_ADD_EXECUTABLE_AND_TEST(
+      Map_LL
+      SOURCES Map.cpp
+      NUM_MPI_PROCS 1
+      COMM serial mpi
+      TARGET_DEFINES -DGALERI_TEST_USE_LONGLONG_GO
+      )
 
-  TRIBITS_ADD_EXECUTABLE_AND_TEST(
-    CrsMatrix_LL
-    SOURCES CrsMatrix.cpp
-    NUM_MPI_PROCS 1
-    COMM serial mpi
-	TARGET_DEFINES -DGALERI_TEST_USE_LONGLONG_GO
-    )
+    TRIBITS_ADD_EXECUTABLE_AND_TEST(
+      CrsMatrix_LL
+      SOURCES CrsMatrix.cpp
+      NUM_MPI_PROCS 1
+      COMM serial mpi
+      TARGET_DEFINES -DGALERI_TEST_USE_LONGLONG_GO
+      )
 
-  TRIBITS_ADD_EXECUTABLE_AND_TEST(
-    LinearProblem_LL
-    SOURCES LinearProblem.cpp
-    NUM_MPI_PROCS 1
-    COMM serial mpi
-	TARGET_DEFINES -DGALERI_TEST_USE_LONGLONG_GO
-    )
+    TRIBITS_ADD_EXECUTABLE_AND_TEST(
+      LinearProblem_LL
+      SOURCES LinearProblem.cpp
+      NUM_MPI_PROCS 1
+      COMM serial mpi
+      TARGET_DEFINES -DGALERI_TEST_USE_LONGLONG_GO
+      )
 
+  ENDIF()
 ENDIF()

